### DISCUSSION TITLE
Invoices

### DISCUSSION THREAD

**Discussion by cedricfortuin** [lunarphp/lunar#1153] [Feature Requests]
It would be awesome to have the ability to do CRUD on invoices, related to customers and orders.

**Comment by glennjacobs** [comment]
Would this not be a duplication of the order data? What requirements did you imagine?

**Comment by cedricfortuin** [comment]
Having used Lightspeed for a while, invoices are used for things like returns to ensure the returned item is paid back to the customer. With that, having an overview of invoices is a requirement for the Dutch government.

My bad if it is requested already tho! :D
